modules/drivers/sqlserver/src/metabase/driver/sqlserver.clj

```diff
@@ -652,8 +652,10 @@
      (with-open [stmt (.prepareStatement conn "SELECT sysdatetimeoffset();")
                  rset (.executeQuery stmt)]
        (when (.next rset)
-         (when-let [offset-date-time (.getObject rset 1 java.time.OffsetDateTime)]
-           (t/zone-offset offset-date-time)))))))
+         (some-> rset
```

Comment:
I think this is the wrong place for the change, as the docstring for `db-default-timezone` Says:
> This method should return a [[String]], a [[java.time.ZoneId]], or a [[java.time.ZoneOffset]].

So I think this should be done right at the `log/info` where it can be humanized no matter which driver returns a `ZoneOffset`.